docs/fieldnorm/life_cycle.md

```diff
@@ -1,7 +1,56 @@
+
+  c.       Utilize a task management system integrated with your Git development system to manage tasks.
+
+  d.       Maintain communication with end users throughout the development life cycle in order to prevent unexpected adjustments to requirements in later stages of development.
+
+A widely adopted software development strategy involves dividing a development project over several periods. It is advised to use an approach that enables to release partial deliverables to users each period, which enables user feedback upon each deliverable during the entire lifetime of the project, and provides ample opportunities for clarification of requirements or adapt their priorities.   
+
+3.      Design
+Here the software architecture is defined, describing the high-level structure, components, interfaces, and interactions of the software system. The architectural design lays the groundwork for the detailed design and implementation phases.
+
+4.      Development
+The development stage will be the big bulk of a project, here code is created that will eventually define the software.
+
+  a.       Usage of a decentralized code management system, such as git, together with a centralised repository, such as Github, Gitlab or Bitbucket, that is seamlessly integrated with a task management system, is strongly recommended.
+
+  b.       Have a coding policy that covers which language is used, how the language is formatted, do’s and don’ts, the use of inline documentation. This policy is not to be strictly enforced, but will open discussions on how the team likes its code and will lead to a more concise and unified coding strategy.
+
+  c.       Write code with unit and integration tests. Moreover, conducting regular code reviews is highly recommended. These steps are crucial for code verification and will be invaluable during software maintenance, testing, and release phases. It enables a wider understanding of parts of the code as unit tests often embody general usage as well as very specific exceptions and code reviews enforce colleagues to have a look and understand what your code is doing.
+
+  d.       Establish a well-defined development and release cycle with set time periods for preparation, development, testing & releasing. This allows multiple developers to work cohesively on the project and creates structured testing and release moments during the implementation and maintenance processes. It also enables ample opportunities for user reflections on the deliverables during the lifetime of project.
+
+5.      Testing
+Before implementation, software should be extensively tested, verified and validated. While ISO 15189 does not provide specific guidelines on how to conduct software validation, it emphasizes the importance of ensuring that software is validated and verified following recognized industry standards. We advise the use of the software validation and verification procedures as described in this guideline.
```

Comment:
Maybe add the part about unit and integration (4c) testing to this section

Reply:
I might do this but I'm waiting for the validation/verification part to be written as this will probably be coverd there as well. therefor I would like to keep it in the development section.